db/migrate/20200511024239_add_position_to_books.rb

```diff
@@ -0,0 +1,11 @@
+class AddPositionToBooks < ActiveRecord::Migration[6.0]
+  def up
+    add_column :books, :position, :integer
+    add_column :books, :hide_from_index, :boolean, default: false
+  end
+
+  def down
+    remove_column :books, :position, :integer
+    remove_column :books, :hide_from_index, :boolean, default: false
+  end
```

Comment:
```suggestion
  def change
    add_column :books, :position, :integer
    add_column :books, :hide_from_index, :boolean, default: false
  end
```

Does this work the same?

Reply:
it does, but it makes no difference